### ISSUE TITLE
[Menu]: a11y: Menu dialog does not have an accessible name

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#5981]
### Describe the bug

[Menu]: a11y: Menu dialog does not have an accessible name

### Reproduction steps

1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/story/modals-popovers-menu--default&globals=backgrounds.grid:!false
2. Open Menu dialog
3. Run axe tool on browser 

Observed error in Axe:  ARIA dialog and alertdialog nodes should have an accessible name

### Expected Behaviour

accessibleName property should be provided.

### UI5 Web Components for React Version

1.29.0

### UI5 Web Components Version

1.24.0

### Browser

Chrome, Edge, Firefox, Safari

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
